src/App.css

```diff
@@ -1,24 +1,171 @@
 .App {
   text-align: center;
 }
+
 .App-header-custom {
   width: auto;
-height: 182px;
-flex-shrink: 0;
-border-radius: 15px;
-background: #A379BC;
+  height: 182px;
+  flex-shrink: 0;
+  border-radius: 15px;
+  background: #A379BC;
+}
+
+.header-content {
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+}
+
+.header-right {
+  display: flex;
+  align-items: center;
+}
+.header-left {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
 }
-.gym-name{
+.box-container {
+  display: flex; /* Add flex display to arrange .all-box and .week-box side by side */
+  align-items: center; /* Center items vertically */
+}
+
+
+/* Default styles for .gym-name */
+.gym-name {
   color: #000;
-font-family: Noto Sans;
-font-size: 20px;
-font-style: normal;
-font-weight: 700;
-line-height: normal;
-position: fixed;
-margin-left: 88px;
-margin-right: auto;
-margin-top:51px ;
-margin-bottom: 141px;
-
-}
\ No newline at end of file
+  font-family: 'Noto Sans', sans-serif;
+  font-size: 20px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: normal;
+  position: fixed;
+  margin-left: 88px;
+  margin-right: auto;
+  margin-top: 51px;
+  margin-bottom: 141px;
+}
+
+/* Adjust styles for screens smaller than 600px */
+@media (max-width: 600px) {
+  .gym-name {
+    font-size: 16px;
+    /* Adjust font size for smaller screens */
+    margin-left: 20px;
+    /* Adjust margins for smaller screens */
+    margin-right: auto;
+    margin-top: 20px;
+    margin-bottom: 40px;
+  }
+}
+
+/* Adjust styles for screens smaller than 400px */
+@media (max-width: 400px) {
+  .gym-name {
+    font-size: 14px;
+    /* Further reduce font size for very small screens */
+    margin-left: 10px;
+    /* Further adjust margins for very small screens */
+    margin-top: 10px;
+    margin-bottom: 20px;
+  }
+}
+
+.centre-text {
+  color: #000;
+  text-align: center;
+  font-family: 'Noto Sans', sans-serif;
+  font-size: 24px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: normal;
+  padding-top: 51px;
+  margin-left: 0%;
+  text-align: center;
+}
+
+.tag-box {
+  display: flex;
+  width: 99px;
+  height: 47px;
+  flex-direction: column;
+  justify-content: center;
+  flex-shrink: 0;
+  background-color: #C7C7C7;
+  border-radius: 8px;
+  margin-right: 20px;
+  margin-bottom: 20px;
+}
+
+.reset-box {
+  display: flex;
+  width: 164px;
+  height: 113px;
+  flex-direction: column;
+  justify-content: center;
+  flex-shrink: 0;
+  background-color: #C7C7C7;
+  border-radius: 8px;
+  margin-right: 20px;
+  margin-bottom: 20px;
+  margin-top: 20px;
+}
+
+.tag-text {
+  display: flex;
+  width: 99px;
```

Comment:
What's your reasoning for specific widths and heights in 'p' element that's embedded in a 'div'?